.circleci/config.yml

```diff
@@ -24,6 +24,22 @@ jobs:
         paths:
         - venv/
         key: python-packages-v1-{{ .Branch }}-{{ checksum "requirements.txt" }}-{{ checksum "constraints.txt" }}
+  verify-generated-sql:
```

Comment:
We'll need to add this to the workflow below:

```
workflows:
  version: 2
  build:
    jobs:
    - build
    - verify-generated-sql
    - deploy:
        requires:
        # can't run in parallel because CIRCLE_BUILD_NUM is same
        - build
        - verify-generated-sql
        filters:
          branches:
            only: master
          tags:
            only: /.*/
```

Reply:
Done.